naff/api/http/http_requests/threads.py

```diff
@@ -189,3 +190,44 @@ async def create_thread(
             payload["type"] = thread_type or ChannelTypes.GUILD_PUBLIC_THREAD
             payload["invitable"] = invitable
             return await self.request(Route("POST", f"/channels/{channel_id}/threads"), data=payload, reason=reason)
+
+    async def create_forum_thread(
+        self,
+        channel_id: "Snowflake_Type",
+        name: str,
+        auto_archive_duration: int,
+        message: dict | FormData,
+        applied_tags: List[str] = None,
```

Comment:
Suppling a list of tags (as str) does not work - discord returns a bad payload error

Should be tested when the feature releases and the API is stable ( :) )